ingest/cell_metadata.py

```diff
@@ -25,6 +25,7 @@ def __init__(self, file_path, file_id: str = None, study_accession: str = None):
         IngestFiles.__init__(self, file_path, self.ALLOWED_FILE_TYPES)
         self.headers = self.get_next_line(
             increase_line_count=False)
+        print(self.headers)
```

Comment:
Is there a reason we're printing these here?

Reply:
No, it was for debugging. Fixed in 6f04851